lang/en.yml

```diff
@@ -2,7 +2,7 @@ en:
   SilverStripe\RealMe\Authenticator\LoginForm:
     ASSERTLOGINBUTTON: 'Share your details with {orgname}'
     AUTHENTICATOR_NAME: 'RealMe Account'
-    LOGINBUTTON: Login
+    LOGINBUTTON: 'Log in'
```

Comment:
Why is this changing? There's a `LOGINBUTTON2: 'Log in'` right below this

Reply:
This is the correct value. "Login" is not correct.